src/engines/html5/html5.js

```diff
@@ -904,7 +904,15 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @return {number[]} - playback rates
    */
   get playbackRates(): Array<number> {
-    return Html5.PLAYBACK_RATES;
+    return this._config.playbackRates;
+  }
+
+  /**
+   * set the playback rates
+   * @param {number[]} playbackRates - playback rates
+   */
+  set playbackRates(playbackRates: Array<number>): void {
+    this._config.playbackRates = playbackRates;
```

Comment:
html should return always `Html5.PLAYBACK_RATES` the overriding should be handled in the player playbackRates getter:
```
if (this._playbackRates) {
  return this._playbackRates;
} else if (this._engine) {
  return this._engine.playbackRates;
}
return [];
```
`this._playbackRates` should be set in `_handlePlaybackOptions`